### ISSUE TITLE
Version module is always 1.0.0

### ISSUE THREAD

**Issue by moobyfr** [neutrinolabs/xorgxrdp#113]
In the Xorg lors, the modules have always the same value 1.0.0.
The version release should be used to get accurate version in the logs

``module/rdp.h `` contains the version info

**Comment by metalefty** [comment]
```
[   660.814] (II) LoadModule: "xorgxrdp"
[   660.815] (II) Loading /usr/local/lib/xorg/modules/libxorgxrdp.so
[   660.898] (II) Module XORGXRDP: vendor="X.Org Foundation"
[   660.898]    compiled for 1.18.4, module version = 1.0.0
[   660.898]    ABI class: X.Org Video Driver, version 20.0
[   660.898] xorgxrdpSetup:
[   660.899] (II) LoadModule: "fb"
[   660.899] (II) Loading /usr/local/lib/xorg/modules/libfb.so
[   661.179] (II) Module fb: vendor="X.Org Foundation"
[   661.179]    compiled for 1.18.4, module version = 1.0.0
[   661.179]    ABI class: X.Org ANSI C Emulation, version 0.4
[   661.179] (II) LoadModule: "xrdpdev"
[   661.237] (II) Loading /usr/local/lib/xorg/modules/drivers/xrdpdev_drv.so
[   661.413] (II) Module XRDPDEV: vendor="X.Org Foundation"
[   661.413]    compiled for 1.18.4, module version = 1.0.0
[   661.413]    ABI class: X.Org Video Driver, version 20.0
[   661.413] xrdpdevSetup:
[   661.413] (II) LoadModule: "xrdpmouse"
[   661.414] (II) Loading /usr/local/lib/xorg/modules/input/xrdpmouse_drv.so
[   661.526] (II) Module XRDPMOUSE: vendor="X.Org Foundation"
[   661.526]    compiled for 1.18.4, module version = 1.0.0
[   661.526]    Module class: X.Org XInput Driver
[   661.526]    ABI class: X.Org XInput driver, version 22.1
[   661.526] rdpmousePlug:
[   661.526] (II) LoadModule: "xrdpkeyb"
[   661.526] (II) Loading /usr/local/lib/xorg/modules/input/xrdpkeyb_drv.so
[   661.692] (II) Module XRDPKEYB: vendor="X.Org Foundation"
[   661.692]    compiled for 1.18.4, module version = 1.0.0
[   661.692]    Module class: X.Org XInput Driver
[   661.692]    ABI class: X.Org XInput driver, version 22.1
[   661.692] rdpkeybPlug:
[   661.692] rdpIdentify:
```

**Comment by metalefty** [comment]
I've created #114. Not replacing it automatically at this time.

**Comment by moobyfr** [comment]
I had to stop this morning, but I saw the actual way for autoconf to magically compute version:

https://lists.gnu.org/archive/html/autoconf/2007-12/msg00027.html
not really pretty, but better with https://www.gnu.org/software/automake/manual/html_node/Rebuilding.html

**Comment by metalefty** [comment]
Okay, I'll try.

**Comment by metalefty** [comment]
I've done #115. A little bit more hacky but the version in configure.ac like `0.2.5` will be split into `0`, `2`, `5` and passed  via command line macro. 

@moobyfr what do you think?

**Comment by metalefty** [comment]
Merged #115.
